core/src/edu/asu/c3simulator/screens/DifficultySelectionScreen.java

```diff
@@ -185,5 +192,5 @@ public void show()
 	{
 		Gdx.input.setInputProcessor(stage);
 	}
-	
+
 }

```

Comment:
In the future, don't commit these changes, unless they are committed as a cleanup commit in their own branch/pull.

Also, it looks like your settings diverged at some point. Indented lines are acceptable. Please update your formatter to comply.